e2e/metrics/cluster_test.go

```diff
@@ -87,6 +88,9 @@ func expectMetrics(name string, expectedMetrics MetricsSelector, expectExist boo
 
 var _ = Describe("Cluster Metrics", Label("cluster"), func() {
 	It("should have metrics for the local cluster resource", func() {
+		if os.Getenv("SHARD") != "" {
```

Comment:
I am not following this... Why not check the `shard` variable directly? Why would this not be set for cluster tests if it is for the whole test suite?